src/core/src/main.rs

```diff
@@ -83,35 +79,31 @@ fn main() {
         .value_of("connect")
         .unwrap_or("tcp://localhost:5050");
 
-    let console_log_level;
+    let log_level;
     match matches.occurrences_of("verbose") {
-        0 => console_log_level = LevelFilter::Warn,
-        1 => console_log_level = LevelFilter::Info,
-        2 => console_log_level = LevelFilter::Debug,
-        3 | _ => console_log_level = LevelFilter::Trace,
+        0 => log_level = LevelFilter::Warn,
+        1 => log_level = LevelFilter::Info,
+        2 => log_level = LevelFilter::Debug,
+        3 | _ => log_level = LevelFilter::Trace,
     }
 
    let stdout = ConsoleAppender::builder()
         .encoder(Box::new(PatternEncoder::new(
-            "{h({l:5.5})} | {({M}:{L}):20.20} | {m}{n}",
+            "{d:22.22} {h({l:5.5})} | {({M}:{L}):30.30} | {m}{n}",
         )))
         .build();
 
-    let requests = FileAppender::builder()
+    let fileout = FileAppender::builder()
         .encoder(Box::new(PatternEncoder::new(
-        "{h({l:5.5})} | {({M}:{L}):20.20} | {m}{n}",
+            "{d:22.22} {h({l:5.5})} | {({M}:{L}):30.30} | {m}{n}",
         )))
         .build("log/debug.log")
-       .unwrap();
+        .expect("Could not build file appender");
 
     let config = Config::builder()
         .appender(Appender::builder().build("stdout", Box::new(stdout)))
-        .appender(Appender::builder().build("requests", Box::new(requests)))
-        .logger(Logger::builder()
-        .appender("requests")
-        .additive(false)
-        .build("app::requests", LevelFilter::Trace))
-        .build(Root::builder().appender("requests").build(console_log_level))
+        .appender(Appender::builder().build("fileout", Box::new(fileout)))
+        .build(Root::builder().appender("stdout").appender("fileout").build(log_level))
```

Comment:
check if you need .appender("stdout").appender("fileout"). since appender is already added in Config::builder()

Reply:
The initial appenders create file/stdout appenders. Whereas the ones in build attach the appender that needs to be activated or attached to the root.